packages/quick_actions/quick_actions_ios/ios/Classes/FLTQuickActionsPlugin_Test.h

```diff
@@ -0,0 +1,20 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import Flutter;
+
+NS_ASSUME_NONNULL_BEGIN
+
+/// APIs exposed for unit tests.
+@interface FLTQuickActionsPlugin (Test)
+
+/// Initializes a FLTQuickActionsPlugin with the given method channel.
+/// API exposed for unit tests.
+/// @param channel A method channel.
+/// @return The initialized FLTQuickActionsPlugin.
+- (instancetype)initWithChannel:(FlutterMethodChannel *)channel;
```

Comment:
Designated initializer?

Reply:
Got this error. Looks like designated initializer does not support class categories: 
```
'objc_designated_initializer' attribute only applies to init methods of interface or class extension declarations
```

Reply:
Oh right